### DISCUSSION TITLE
wifi not connect

### DISCUSSION THREAD

**Discussion by big-vl** [bassamanator/armbian-orangepi-how-to#4] [General]
Wifi doesn't work, won't connect on minimal image

**Comment by bassamanator** [comment]
What device are you using?

I'm using the minimal image myself for my OPiZ2W, and the wifi does work, however, I started off with `Armbian_community_24.8.0-trunk.314_Orangepizero2w_bookworm_current_6.6.36_minimal.img.xz`.

As you can see [here](https://github.com/armbian/build/blob/main/extensions/preset-firstrun.sh), the same `preset`s still apply, so the method should still work. If it isn't, then you have to create an issue for `Armbian` proper.
